gui/slick/js/manageEpisodeStatuses.js

```diff
@@ -1,73 +1,96 @@
+	});
 
-    $('.get_more_eps').click(function(){
-        var cur_indexer_id = $(this).attr('id');
-        var checked = $('#allCheck-' + cur_indexer_id).prop('checked');
-        var last_row = $('tr#' + cur_indexer_id);
-        
-        $.getJSON(sbRoot + '/manage/showEpisodeStatuses',
-                  {
-                   indexer_id: cur_indexer_id,
-                   whichStatus: $('#oldStatus').val()
-                  },
-                  function (data) {
-                      $.each(data, function(season,eps){
-                          $.each(eps, function(episode, name) {
-                              //alert(season+'x'+episode+': '+name);
-                              last_row.after(make_row(cur_indexer_id, season, episode, name, checked));
-                          });
-                      });
-                  });
-        $(this).hide();
-    });
+	$('.get_more_eps').click(function(){
+		var cur_indexer_id = $(this).attr('id');
+		var checked = $('#allCheck-' + cur_indexer_id).prop('checked');
+		var last_row = $('tr#' + cur_indexer_id);
+		
+		$.getJSON(sbRoot + '/manage/showEpisodeStatuses',
+				  {
+				   indexer_id: cur_indexer_id,
+				   whichStatus: $('#oldStatus').val()
+				  },
+				  function (data) {
+					  $.each(data, function(season,eps){
+						  $.each(eps, function(episode, name) {
+							  //alert(season+'x'+episode+': '+name);
+							  last_row.after(make_row(cur_indexer_id, season, episode, name, checked));
+						  });
+					  });
+				  });
+		$(this).hide();
+	});
 
-    // selects all visible episode checkboxes.
-    $('.selectAllShows').click(function(){
-        $('.allCheck').each(function(){
-                this.checked = true;
-        });
-        $('input[class*="-epcheck"]').each(function(){
-                this.checked = true;
-        });
-    });
+	$('.expandAll').click(function() {
+		$('.get_more_eps').each(function() {
+			var cur_indexer_id = $(this).attr('id');
+			var checked = $('#allCheck-' + cur_indexer_id).prop('checked');
+			var last_row = $('tr#' + cur_indexer_id);
 
-    // clears all visible episode checkboxes and the season selectors
-    $('.unselectAllShows').click(function(){
-        $('.allCheck').each(function(){
-                this.checked = false;
-        });
-        $('input[class*="-epcheck"]').each(function(){
-                this.checked = false;
-        });
-    });
+			$.getJSON(sbRoot + '/manage/showEpisodeStatuses',
+					  {
+					   indexer_id: cur_indexer_id,
+					   whichStatus: $('#oldStatus').val()
+					  },
+					  function (data) {
+						  $.each(data, function(season, eps) {
+							  $.each(eps, function(episode, name) {
+								  last_row.after(make_row(cur_indexer_id, season, episode, name, checked));
+							  });
+						  });
+					  });
+			$(this).hide();
+		});
+		$(this).hide();
+	});
+
+	// selects all visible episode checkboxes.
+	$('.selectAllShows').click(function(){
+		$('.allCheck').each(function(){
```

Comment:
this can be simplified...
from ... $('.allCheck').each(function(){
to  ... $('.allCheck, input[class*="-epcheck"]').each(function(){
and then removing the each block at L#81